src/data/actions/productsListAction.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-undef */
+import axios from 'axios';
+import { GET_PRODUCTS_LIST_SUCCESS } from '../types/types';
+
+export const getProductsList = () => (dispatch) => axios.get('http://3.134.102.30/products/list')
+  .then((res) => {
```

Comment:
is it possible to refactor the API call to utilize the shared api file?  Is there a good reason to separate the API calls to multiple files?

Reply:
I'll see if I can utilize the API calls you setup in a bit. I separated them into different files so it might be cleaner in the end. I'm not sure how much else we'll need to add to each of these files as things progress. If they're looking small enough I'm okay with combing them.

Reply:
I can't say exactly if there is a "best" way to accomplish this (combined into one file or separated into multiple components).  It seems like having them all in one place and then having them all dump into store at once would be fine, but Teddi did say that technically we're all supposed to be doing our own calls for our own component and everything is supposed to be in it's own specialized component in react so there are good arguments for both methods.

Either way, if we have one method working and that method doesn't have any known future conflicts, I think we should move forward and keep our decision here in mind just incase we have to come back and switch to the other method.